data/tlg2586/tlg002/tlg2586.tlg002.1st1K-grc1.xml

```diff
@@ -0,0 +1,2063 @@
+                    <resp>Editor-in-Chief, Perseus Digital Library</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Matt Munson</persName>
+                    <resp>Project Manager (University of Leipzig), 2016 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Annette Gessner</persName>
+                    <resp>Project Assistant (University of Leipzig) 2015 - present</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Thibault Clérice</persName>
+                    <resp>Lead Developer (University of Leipzig)</resp>
+                </respStmt>
+                <respStmt>
+                    <persName>Bruce Robertson</persName>
+                    <resp>Technical Advisor</resp>
+                </respStmt>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Harvard College Library</authority>
+                <idno type="filename">tlg2586.tlg002.1st1K-grc1.xml</idno>
+                <availability>
+                    <p n="">Available under a Creative Commons Attribution-ShareAlike 4.0
+                        International License</p>
+                </availability>
+                <date>2017</date>
+                <publisher>Harvard College Library</publisher>
+                <pubPlace>United States</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Rhetores Graeci</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Leonardi Spengel</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg2586">Menander</author>
+                            <imprint>
+                                <publisher>Teubner</publisher>
+                                <pubPlace>Leipzig</pubPlace>
+                                <date>1856</date>
+                            </imprint>
+                            <biblScope unit="volume">3</biblScope>
+                        </monogr>
+                        <ref
+                            target="https://archive.org/stream/bub_gb_g31KAAAAYAAJ#page/n407/mode/2up"
+                            >Internet Archive</ref>
+                    </biblStruct>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p n="">Text encoded in accordance with the latest EpiDoc standards</p>
+            <refsDecl n="CTS">
+                <cRefPattern matchPattern="(.+).(.+)" n="paragraph"
+                    replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:p[@n='$2'])"/>
```

Comment:
I have discovered that it is not such a good idea to use `<p>` elements as citation units. What happens is that any `<p>`s that don't have an `n` attribute are then left out of the text (I need to go back and correct the ones that are like that). So I would suggest just enclosing the `<p>` elements in `<div>`s instead.